src/main/java/org/cactoos/http/HtContentType.java

```diff
@@ -40,7 +42,7 @@
  * @version $Id$
  * @since 0.1
  */
-public final class HtContentType implements Scalar<String> {
+public final class HtContentType implements Scalar<List<String>> {
```

Comment:
@krzyk just want to note that this is kind of weird, but I guess that nothing stops a sender to insert multiple `Content-Type` headers.